app/routes/application.js

```diff
@@ -48,5 +49,6 @@ export default class ApplicationRoute extends Route {
     if (this.session.isAuthenticated && !this.currentSession.hasAccessToApplication) {
       this.router.transitionTo('accountless-users');
     }
+    this.transitionHistory.registerTransition();
```

Comment:
Ember has built-in events for doing actions on route changes. Check [this for an example in our code](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/app/services/plausible.js#L14-L27) and [this guide from Ember](https://guides.emberjs.com/release/routing/preventing-and-retrying-transitions/)